src/models/LocalDataModel.ts

```diff
@@ -0,0 +1,12 @@
+import { observable, action } from "mobx";
+import { IDataModel } from "./i_dataModel";
+
+export class LocalDataModel implements IDataModel {
+    save(jsonData: string) {
+        localStorage.setItem("AppData", jsonData);
```

Comment:
this storage is designed to work only with the "AppData" key. Would be nice to have a possibility to pass a key as well